hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java

```diff
@@ -386,7 +386,7 @@ public String getQuorumName() {
         return recordFactoryConstructor;
     }
 
-    public QueryableEntry newQueryEntry(Data key, Object value) {
+    public QueryableEntryImpl newQueryEntry(Data key, Object value) {
```

Comment:
The quaryable entry gives me sleepless nights. But now we have an abstraction for this ugly thing?